src/docs/development/packages-and-plugins/plugin-api-migration.md

```diff
@@ -11,7 +11,7 @@ and redirect to this file.
 
```

Comment:
we should qualify that this is for Android in the permalink, title etc.

Reply:
done